index.html

```diff
@@ -14,5 +15,24 @@ <h4 class="nav-logo">nathaniel</h4>
 				<a class="nav-link" href="#contact">Contact</a>
 			</div>
 		</nav>
+		<main>
+			<div class="main-content">
+				<img />
```

Comment:
I'm guessing this is a placeholder for an img?

Reply:
mortified that I left this out, filling in now!